### ISSUE TITLE
Glide still depends on Support Libs so Jetifier Cannot be Turned Off

### ISSUE THREAD

**Issue by eygraber** [bumptech/glide#3788]
According to https://github.com/plnice/can-i-drop-jetifier, Glide still references support libs and Jetifier cannot be turned off.

```
* com.github.bumptech.glide:glide:4.9.0
  \-- com.android.support:support-fragment:27.1.1
  \-- com.android.support:animated-vector-drawable:27.1.1
  \-- com.github.bumptech.glide:gifdecoder:4.9.0
   \-- com.android.support:support-annotations:27.1.1
```

**Comment by sjudd** [comment]
Duplicate of #3184

**Comment by ZakTaccardi** [comment]
@sjudd seems to me that this needs to be re-opened?

**Comment by TWiStErRob** [comment]
@ZakTaccardi https://github.com/bumptech/glide/pull/3678/files?file-filters%5B%5D=.gradle#diff-0d1d9100a9b6813b8cb5c470bf313d00

AndroidX support is already merged and 4.10-SNAPSHOT should contain no support libs. Why are you suggesting to reopen?

**Comment by ZakTaccardi** [comment]
I misread the issue - I thought it was referencing the 4.10 snapshot.

Hoping for a 4.10 release soon!
